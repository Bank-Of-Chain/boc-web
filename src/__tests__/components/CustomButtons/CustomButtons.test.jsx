import React from 'react'
import { render } from '@testing-library/react'
import CustomButtons from '@/components/CustomButtons/Button'
import { describe, expect, it } from 'vitest'

describe('xxxxxxxx', () => {
  it('CustomButtons Component Render', () => {
    const { asFragment } = render(<CustomButtons className="abc" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
