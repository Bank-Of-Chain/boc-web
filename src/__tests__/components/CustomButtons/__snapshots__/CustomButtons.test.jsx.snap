// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`xxxxxxxx > CustomButtons Component Render 1`] = `
<DocumentFragment>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-1 abc"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiButton-label"
    />
    <span
      class="MuiTouchRipple-root"
    />
  </button>
</DocumentFragment>
`;
